opm/simulators/flow/ReservoirCouplingMaster.hpp

```diff
@@ -61,9 +54,10 @@ class ReservoirCouplingMaster {
 
     const Parallel::Communication &comm_;
     const Schedule& schedule_;
-    // MPI communicators for the slave processes
-    std::vector<MPI_Comm_Ptr> master_slave_comm_;
+    std::size_t num_slaves_ = 0;  // Initially zero, will be updated in spawnSlaveProcesses()
+    std::vector<MPI_Comm_Ptr> master_slave_comm_; // MPI communicators for the slave processes
```

Comment:
Is the documentation string correct? I thought this would be an intercommunicator and would inlude all ranks from the original one and the ones from the newly created one ("Intercommunicator between original group and the newly spawned group (handle)." says the documentation)

Reply:
We currently use these handles to communicate (MPI_Send(), MPI_Recv()) between a specific rank in the master and a specific rank in the slave process. So the comment should maybe use "to" instead of "for" ?  Like this: "MPI communicators to the slave processes" ?